en/python_installation/instructions.md

```diff
@@ -52,10 +52,10 @@ It is very likely that you already have Python installed out of the box. To chec
 {% filename %}command-line{% endfilename %}
 ```
 $ python3 --version
-Python 3.6.1
+Python {{ book.py_release }}
 ```
 
-If you have a different version of Python installed, at least 3.4.0 (e.g. 3.6.0), then you don't have to upgrade. If you don't have Python installed, or if you want a different version, first check what Linux distribution you are using with the following command:
+If you have a different version of Python installed, at least 3.4.0 (e.g. {{ book.py_release }}), then you don't have to upgrade. If you don't have Python installed, or if you want a different version, first check what Linux distribution you are using with the following command:
```

Comment:
[Django 3.1 requires Python 3.6 or greater](https://docs.djangoproject.com/en/3.1/faq/install/#faq-python-version-support).
```suggestion
If you have a different version of Python installed, at least 3.6.0 (e.g. {{ book.py_release }}), then you don't have to upgrade. If you don't have Python installed, or if you want a different version, first check what Linux distribution you are using with the following command:
```

(Maybe we should introduce a variable for the minimum Python version, too?)

Reply:
That's a great idea. How does the `{{ book.py_min_version }}` sounds?

Reply:
@das-g Check commit https://github.com/DjangoGirls/tutorial/pull/1683/commits/d90b4a38a4531b28701c866cff611f1cc7c8161e